src/types/SELECT.ts

```diff
@@ -0,0 +1,5 @@
+export enum SELECT {
```

Comment:
Consider renaming the enum to 'Select' to follow the naming convention of PascalCase for enums.